admin/acceptance.py

```diff
@@ -404,14 +409,22 @@ def main(reactor, args, base_path, top_level):
 
     try:
         nodes = yield runner.start_nodes(reactor)
+
+        ca_directory = FilePath(mkdtemp())
+        print("Generating certificates in: {}".format(ca_directory.path))
+        certificates = Certificates.generate(ca_directory, nodes[0].address,
+                                             len(nodes))
+
```

Comment:
The acceptance tests don't need a new certificate every time, I bet.  It would be nice to re-use existing certificates if they exist since key generation is time consuming.